lib/gui/modules/analytics.js

```diff
@@ -164,23 +87,39 @@ analytics.service('AnalyticsService', function($log, $window, $mixpanel, Setting
    * @public
    *
    * @description
-   * This function logs an exception in TrackJS.
+   * This function logs an exception to error reporting services.
    *
    * @param {Error} exception - exception
    *
    * @example
    * AnalyticsService.logException(new Error('Something happened'));
    */
   this.logException = (exception) => {
-    if (_.every([
-      SettingsModel.get('errorReporting'),
-      isRunningInAsar(),
-      errors.shouldReport(exception)
-    ])) {
-      $window.trackJs.track(exception);
-    }
+    resinCorvus.logException(exception);
+  };
 
-    $log.error(exception);
+  /**
+   * @summary Enable analytics
+   * @function
+   * @public
+   *
+   * @example
+   * AnalyticsService.enable()
+   */
+  this.enable = () => {
```

Comment:
Also, can we ensure that if `resinCorvus` is enabled by default, and the user has disabled error reporting in the settings, then we don't send *any* analytics until ensuring that's what the user wants?

I think that other code might be evaluated before this, causing the startup events to be sent before we realise that analytics reporting shouldn't happen.